app/models/user.rb

```diff
@@ -0,0 +1,14 @@
+class User < ApplicationRecord
+  # Include default devise modules. Others available are:
+  # :confirmable, :lockable, :timeoutable and :omniauthable
+  devise :database_authenticatable, :registerable,
+         :recoverable, :rememberable, :trackable, :validatable
+  validates :name, :surname, presence: true,
+                             length: { in: 3..30 },
```

Comment:
Use two spaces per indentation level

Reply:
I don't have to use here 2 spaces per indentation level

Reply:
Are you sure?)

Reply:
yes, I'm )

Reply:
Is there any rule about that in Ruby/Rails styleguides?

Reply:
i have not seen any rule in styleguides